Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -52,10 +46,26 @@ static DWRITE_TEXT_ALIGNMENT __CoreTextAlignmentToDwrite(CTTextAlignment alignme
     }
 }
 
+static DWRITE_WORD_WRAPPING __CTLineBreakModeToDWrite(CTLineBreakMode lineBreakMode) {
```

Comment:
you used the word export there and I want to make sure we actually know what the relationships are there. Did you mean exported, as in a dll export, or extern, as in visible outside the current translation unit (.mm or .cpp)

---
In reply to: [90746929](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90746929) [](ancestors = 90746929,90742638)